texts/smlm-lab-course/smlm-lab-manual.tex

```diff
@@ -368,6 +368,25 @@ \subsection{Photoswitching vs Photoactivation}
     \label{table:photophysics-comparison}
 \end{table}
 
+\section{The Role of the Buffer}
+
+The STORM buffer used in this course contains the following components.
+
+\begin{itemize}
+    \item Trizma base
+    \item NaCl
+    \item Glucose
+    \item Monoethanolamine (commonly called MEA)
```

Comment:
95% chance this is semantics but its always been called Cysteamine to me. probably just another name but i haven't heard it called monoethanolamine

Reply:
When I looked online just now, cystamine is betamercaptethanol. 🤷‍♂️

Reply:
I used 2-aminoethanol based on Laila's recommendation. Let me know if I need to change it 🚀